src/LendingPool.sol

```diff
@@ -887,6 +887,57 @@ contract LendingPool is LendingPoolGuardian, TrustedCreditor, DebtToken, Interes
         //Event emitted by Liquidator.
     }
 
+    function settleLiquidation_NEW(
+        address account,
+        address originalOwner,
+        uint256 badDebt,
+        address initiator,
+        uint256 liquidationInitiatorReward,
+        address terminator,
+        uint256 auctionTerminationReward,
+        uint256 liquidationFee,
+        uint256 remainder
+    ) external onlyLiquidator processInterests {
+        //Make Initiator rewards claimable for liquidationInitiator[account].
+        realisedLiquidityOf[initiator] += liquidationInitiatorReward;
+
+        if (badDebt > 0) {
+            if (badDebt < liquidationFee + auctionTerminationReward) {
```

Comment:
Isn't the payout of the `liquidationFee` and `auctionTerminationReward ` missing here?